apps/ingest/src/ingest.ts

```diff
@@ -334,7 +334,7 @@ const setAssessmentDue = async (lapsePeriodMonths: number) => {
 }
 
 const fetchStudies = async function* (url: string, username: string, password: string) {
-  const pageSize = 1000
+  const pageSize = 250
```

Comment:
Ideally this would be optionally configurable.

https://github.com/PA-NIHR-CRN/sponsor-engagement-web/blob/1e58cae58c1328b36d997a5e5a4cf521f0cc2fbb/apps/ingest/src/ingest.ts#L427

Reply:
I had thought about this but as the env variables are handled in the task definition, it requires a redeploy to update or manually updating the definition, I think this is part of a bigger piece to move SE secrets to be fetched by the application on initial load which will allow the config to be a bit more flexible/useful